### PR TITLE
Fix missing colours on windows

### PR DESCRIPTION
**Pull request by klauspost**
## Description

#4705 removed all color from Windows output.

## Motivation and Context

The world should have color diversity.

## How to test this PR?

Run any command on Windows. Observe greyness.

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:
- [x] Fixes a regression #4705

### CODE REVIEW

**Review by harshavardhana — APPROVED** [review]

**Review by vadmeste — APPROVED** [review]
LGTM

File: cmd/main.go [hunk 1]
```diff
@@ -116,9 +116,9 @@ func Main(args []string) error {
 	probe.SetAppInfo("Commit", ShortCommitID)
 
 	// Fetch terminal size, if not available, automatically
-	// set globalQuiet to true.
+	// set globalQuiet to true on non-window.
 	if w, h, e := term.GetSize(int(os.Stdin.Fd())); e != nil {
-		globalQuiet = true
+		globalQuiet = runtime.GOOS != "windows"
```
Comment (vadmeste) [inline cmd/main.go:121]: @klauspost https://cs.opensource.google/go/x/term/+/refs/tags/v0.13.0:term_windows.go, it looks like the term package we are using supports Windows; I am not sure what is going on

Comment (harshavardhana) [reply cmd/main.go:121]: May be @klauspost has one of those special windows versions :-)

Comment (klauspost) [reply cmd/main.go:121]: @vadmeste I don't know. It fails on mine.

**Pull request merged**